strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/CronJobDefinition.java

```diff
@@ -12,13 +12,13 @@
 public class CronJobDefinition
 {
 
-    private String jobClass;
+    private Class<?> jobClass;
```

Comment:
@fuss86 can you please check that all this `String` to `Class` refactoring will be safety as mentioned above?

Reply:
All `Acceptance Test` passed, what are your doubts ?